code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm

```diff
@@ -2,9 +2,32 @@
 	name = "explosive spider"
 	icon_state = "spiderling_explosive"
 	spider_price = 40
+	var/devastation_range = -1
+	var/heavy_range = 0
+	var/weak_range = 2
+	var/flash_range = 6
+	var/det_time = 20
 
 /obj/item/weapon/implant/carrion_spider/explosive/activate()
 	..()
-	visible_message(SPAN_DANGER("[src] explodes!"))
-	explosion(get_turf(src), -1, 0, 2, 6)
+	var/obj/item/weapon/implant/carrion_spider/explosive/biteszadusto = new/obj/item/weapon/implant/carrion_spider/explosive(get_turf(src))
```

Comment:
I would just move the current implant to the turf instead of making a new one.

Reply:
some pointers would be nice, not all that familiar with all the basic proc and function yet

Reply:
Eris itself doesn't have any documentation, so sadly the only way to learn is trial and error. It's normal do not do everything perfectly the first try. I don't know the proc to detatch implants from the top of my head, but cruciform or surgery code should contain it.